### ISSUE TITLE
floccus ignores browser-specific links

### ISSUE THREAD

**Issue by V00id** [floccusaddon/floccus#1618]
### Which version of floccus are you using?

5.1.0

### How many bookmarks do you have, roughly?

250

### Are you using other means to sync bookmarks in parallel to floccus?

No

### Sync method

Google Drive

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Edge 124.0.2478.105

### Describe the Bug

Hello,
floccus ignores local Links like "edge://downloads/all".
I've uploaded all my bookmarks into the file on Google Drive and works fine with the exception, that all my "edge" BMs are vanished.

Thanks in advanced.

Regards, V00id

### Expected Behavior

Local Edge BMs are saved as well.

### To Reproduce

Just add a BM, for example "edge://downloads/all" and  then upload to remote.
Delete local BMs.
Then download BMs from remote back to local.
No "edge://" BM.

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
This is somewhat by design as not all browsers support edge links.

**Comment by V00id** [comment]
> This is somewhat by design as not all browsers support edge links.

Hello,
thanks for your immediate answer.

What you are saying is, that not floccus is refusing to transmit the links into the "bookmarks.xbel" file. The firefox browser is filtering them out at import time?

**Comment by marcelklehr** [comment]
No,  sorry, floccus is filtering them, because creating them on Firefox would cause an error. But I suppose, it would be possible to filter them on firefox instead and allow adding them to the xbel file...

**Comment by V00id** [comment]
> No, sorry, floccus is filtering them, because creating them on Firefox would cause an error. But I suppose, it would be possible to filter them on firefox instead and allow adding them to the xbel file...

Hm, the reason why anyone is using a bookmark managing addon is to exchange ONE bookmark collection for multiple browsers. So nothing should be filtered out.

It is the problem of the user, if he don't get it, that he should not use those Edge-Link in Firefox.

PS: Don't get me wrong, I find your floccus addon very good, and plan to donate and use it.
I just want to point out this issue here.

**Comment by marcelklehr** [comment]
Any other schemes than edge:// ?

**Comment by V00id** [comment]
> Any other schemes than edge:// ?

Hello, I dont use Edge anymore, only Firefox. So, I dont need any Bookmark sync addons anymore.
You may close that issue.
